common-util/src/main/kotlin/com/google/devtools/ksp/common/ResourceUtils.kt

```diff
@@ -0,0 +1,28 @@
+package com.google.devtools.ksp.common
+
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.io.InputStream
+
+fun Iterable<File>.getResource(path: String): InputStream? = firstNotNullOfOrNull { root ->
```

Comment:
TODO: Add a test that verifies that you cannot access things in parent directories.